code/game/vehicles/spacepods/spacepod.dm

```diff
@@ -250,9 +250,12 @@
 
 /obj/spacepod/attackby(obj/item/W as obj, mob/user as mob, params)
 	if(iscrowbar(W))
-		hatch_open = !hatch_open
-		playsound(loc, 'sound/items/Crowbar.ogg', 50, 1)
-		to_chat(user, "<span class='notice'>You [hatch_open ? "open" : "close"] the maintenance hatch.</span>")
+		if(!equipment_system.lock_system || allow2enter || hatch_open)
```

Comment:
this would be better around as 
```dm
if(!allow2enter)
    to_chat(user, "<span class='warning'>The hatch is locked shut!</span>")
else
    etc
```